job-descriptions/software-engineer-security.md

```diff
@@ -2,28 +2,31 @@
 
 # Software Engineer - Security
 
-We are looking for an engineer who specializes in security to help us proactively secure our product. This includes the security of Sourcegraph Cloud as well as making it easy for our customers to secure their own private Sourcegraph instances that they deploy on-premise. You will be one of the first security hires at Sourcegraph and will have the opportunity to build a world-class security culture and team from the ground up.
+We are looking for an engineer who is passionate about security, and wants to proactively secure our product and our customers.  You will help our customers secure their on-premise versions of Sourcegraph, and contribute to the security of our product and Sourcegraph cloud.  You will be one of the early security hires at Sourcegraph, helping to build a world-class security team, where everyone contributes, from writing code to running vulnerability scans.
```

Comment:
I am a bit worried that the first two sentences awkwardly have two "ands." Can you think of a way to avoid this without losing the meaning?

Reply:
I struggled with that - a lot.  The best version I hit on is below, but I'd love something more clean.  The first sentence is particularly troublesome.

We are looking for an engineer who is passionate about security, wanting to proactively secure our product and our customers.  You will help our customers secure their on-premise versions of Sourcegraph, while contributing to the security of our product and Sourcegraph cloud.